src/window/city.c

```diff
@@ -18,6 +18,7 @@
 #include "map/bookmark.h"
 #include "map/grid.h"
 #include "scenario/criteria.h"
+#include "scenario/building.h"
```

Comment:
Alphabetical order please :)
```suggestion
#include "scenario/building.h"
#include "scenario/criteria.h"
```